zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/remote/RemoteInterpreterServer.java

```diff
@@ -140,10 +136,9 @@ public static void main(String[] args)
 
 
   @Override
-  public void createInterpreter(String interpreterGroupId, String className, Map<String, String>
-          properties)
-      throws TException {
-
+  public void createInterpreter(String interpreterGroupId, String noteId, String
```

Comment:
@Leemoonsoo Thanks for letting me know. Actually, it's one of points to use noteId - but I mis-point it - and I really want to pass InterpreterContext in case of open and close. how about it?